src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter(Boolean)
+    .forEach((line) => {
+      const [key, value] = line.split(':').map((item) => {
+        const trimValue = item.trim();
+        const removeSemiColumn = trimValue.split(';').join('');
```

Comment:
The operation `trimValue.split(';').join('')` is unnecessary because the string has already been split by semicolons in line 12. This line can be removed to simplify the code.